libs/angular/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "src/",
```

Comment:
Can this be removed? It differs between this file and `tsconfig.lib.json`, and I think it should be safe to remove from all our tsconfig files in this project. It should be inferred from the `include` array when `tsconfig.lib.json` is used in `project.json`.
 
```suggestion
```

Reply:
Yes, good catch 👌🏻